### ISSUE TITLE
Not possible to import .lif files into ImageJ

### ISSUE THREAD

**Issue by Rdornier** [GReD-Clermont/simple-omero-client#54]
Hello @ppouchin,

I have an issue when I try to read .lif images with your client.
The command `imageWrapper.toImageJ()` fails at  `setCalibration(pixels, calibration);` in the version 5.10.0 (it was working fine in the 5.9.1 version).

Edit : I've just tested with the version 5.12.1 and it still fails with the same exception

Here is the exception : 
```

ImageJ 2.3.0/1.53f51; Java 1.8.0_332 [64-bit]; Windows 10 10.0; 225MB of 4539MB (4%)
 
java.lang.RuntimeException: java.util.concurrent.ExecutionException: java.lang.RuntimeException: Module threw exception
	at net.imagej.legacy.LegacyService.runLegacyCompatibleCommand(LegacyService.java:307)
	at net.imagej.legacy.DefaultLegacyHooks.interceptRunPlugIn(DefaultLegacyHooks.java:166)
	at ij.IJ.runPlugIn(IJ.java)
	at ij.Executer.runCommand(Executer.java:150)
	at ij.Executer.run(Executer.java:68)
	at java.lang.Thread.run(Thread.java:750)
Caused by: java.util.concurrent.ExecutionException: java.lang.RuntimeException: Module threw exception
	at java.util.concurrent.FutureTask.report(FutureTask.java:122)
	at java.util.concurrent.FutureTask.get(FutureTask.java:192)
	at net.imagej.legacy.LegacyService.runLegacyCompatibleCommand(LegacyService.java:303)
	... 5 more
Caused by: java.lang.RuntimeException: Module threw exception
	at org.scijava.module.ModuleRunner.call(ModuleRunner.java:127)
	at org.scijava.module.ModuleRunner.call(ModuleRunner.java:63)
	at org.scijava.thread.DefaultThreadService.lambda$wrap$2(DefaultThreadService.java:225)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	... 1 more
Caused by: java.lang.NullPointerException
	at fr.igred.omero.meta.PlaneInfoWrapper.lambda$getMinPosition$0(PlaneInfoWrapper.java:146)
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1384)
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:482)
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:472)
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:708)
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:566)
	at fr.igred.omero.meta.PlaneInfoWrapper.getMinPosition(PlaneInfoWrapper.java:147)
	at fr.igred.omero.repository.PixelsWrapper.getPositionZ(PixelsWrapper.java:272)
	at fr.igred.omero.repository.ImageWrapper.setCalibration(ImageWrapper.java:182)
	at fr.igred.omero.repository.ImageWrapper.toImagePlus(ImageWrapper.java:587)
	at fr.igred.omero.repository.ImageWrapper.toImagePlus(ImageWrapper.java:535)
	at ch.epfl.biop.retrievers.OMERORetriever.getImage(OMERORetriever.java:116)
	at ch.epfl.biop.processing.Processing.run(Processing.java:22)
	at ch.epfl.biop.command.ArgoLightCommand.run(ArgoLightCommand.java:83)
	at org.scijava.module.ModuleRunner.run(ModuleRunner.java:163)
	at org.scijava.module.ModuleRunner.call(ModuleRunner.java:124)
	... 6 more

```

**Comment by ppouchin** [comment]
Ok, it's a NullPointerException coming from the part where I compute the min position. This LIF file may not have the metadata, and OMERO returns null somehow somewhere.
Could you send me a small file with this problem so I can replicate it and fix it? 
I suppose I just need to filter out null values before converting it to `double` but I'd like to see the exact behavior.

**Comment by Rdornier** [comment]
Yes, here is a file.
[argolarge_20x_z1_bis.zip](https://github.com/GReD-Clermont/simple-omero-client/files/10941122/argolarge_20x_z1_bis.zip)

**Comment by ppouchin** [comment]
Should be fixed for version 5 in #55 in release [5.12.2](https://github.com/GReD-Clermont/simple-omero-client/releases/tag/5.12.2)

**Comment by Rdornier** [comment]
Yes, it works.
Thanks !
